content/docs/working_at_the_turing/reasonable_adjustments.md

```diff
@@ -0,0 +1,24 @@
+---
+# Page title as it appears in the navigation menu
+title: "Reasonable Adjustments"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 4
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Reasonable Adjustments Policy
+
+The Alan Turing Institute is committed to creating an inclusive, barrier-free workplace, recognising that providing reasonable adjustments is essential to supporting colleagues in fully participating in their roles.
+A summary of the Institute's reasonable adjustment policy can be found [here](https://www.turing.ac.uk/reasonable-adjustments-policy),
+with the full policy available for internal staff on [Mathison](https://thealanturininstitute.sharepoint.com/sites/gc/Public%20Documents/Corporate%20Governance/Policies/People%20Policies/Reasonable%20Adjustment%20Policy%20Final.pdf).
+
+The policy outlines the Institute’s legal obligations and emphasises a proactive approach to making reasonable adjustments, considering each request carefully even if it falls outside of legal requirements.
+It covers adjustments to workspaces, practices, roles, and other areas where barriers may exist.
+Examples include adjustments for mental health, neurodiverse conditions, visual and hearing impairments, chronic illnesses, and mobility conditions.
```

Comment:
I guess the examples are the part that would be really useful to add. Is it still worth checking with someone from HR as to the level of detail we can have here?

Reply:
Yes, definitely worth asking. I will drop them a line 👍

Reply:
Thanks for flagging @phinate. Will add that in.

Reply:
I think the examples shouldn't preclude us merging this, though, the detail can always be a separate PR :)

Reply:
@phinate Do we know how to make an Access to Work request in the Turing? This doesn't seem to be documented on Mathison......

Reply:
It's not afaik -- i guess it's a process external to the Turing, so I don't know if there's much to do beyond signposting instructions?

Reply:
Oh, I see, that makes sense. Still would be nice to know if there's somebody at the Turing who can support the application / act as a contact. Sounds like the job of HR advisory team, but I have no clue.

I think this might be a wider problem that isn't our job to solve in this PR though.

Reply:
Agreed. It was mentioned in one of the EDI professional development courses if memory serves correctly. 
My email to HR was forwarded on to the advisory team so I will ask them when I hear back.